docs/sources/outgoing-webhooks/_index.md

```diff
@@ -7,7 +7,7 @@ keywords:
   - on-call
   - amixr
   - webhooks
-title: Configure outgoing webhooks for Grafana OnCall
+title: Outgoing Webhooks
```

Comment:
Making the page name a bit shorter:

<img width="313" alt="Screenshot 2023-07-18 at 18 33 32" src="https://github.com/grafana/oncall/assets/20116910/d2020a0b-3670-494b-a5ba-73ac935884a0">